packages/runtime/src/api/resource-types.ts

```diff
@@ -1,10 +0,0 @@
-import { z } from 'zod'
-
-const pagePathnameSliceSchema = z.object({
-  id: z.string(),
-  pathname: z.string(),
-  localizedPathname: z.string().optional().nullable(),
-  __typename: z.literal('PagePathnameSlice'),
-})
-
-export type PagePathnameSlice = z.infer<typeof pagePathnameSliceSchema>
```

Comment:
This was replaced by a simpler extension of the `PagePathnameSliceFragment` definition in `api/types.ts` below.

Reply:
Are the types equivalent?

Reply:
No, the new type is more correct, the Zod type above is actually missing the `basePageId` prop.